configure

```diff
@@ -856,230 +856,6 @@ then
     putvar CFG_LOCAL_RUST_ROOT
 fi
 
-# Force bitrig to build with clang; gcc doesn't like us there
```

Comment:
Although actually, could we avoid all this movement for now? I think the logic here is actually that if any triple is *not* MSVC then we need to look for gcc, but this looks like the logic is that if any triple is MSVC then we don't require gcc